packages/terra-clinical-header/src/themes/clinical-lowlight-theme/Header.module.scss

```diff
@@ -0,0 +1,10 @@
+:local {
```

Comment:
@mjhenkes what are your thoughts on organizing themeing this way so file names are cleaner. If a component has multiple files, we can following the File naming like:
```
Main.jsx
Main.module.scss
Another.jsx
Another.module.scss
/themes
   /some-theme
       Main.moudle.scss
       Another.module.scss
```

Reply:
I could see it, adding the name to every file is a bit annoying. What if we dropped the theme folder?:
```
Main.jsx
Main.module.scss
Another.jsx
Another.module.scss
 /some-theme
     Main.moudle.scss
     Another.module.scss
```

Reply:
I'm fine with dropping the theme folder. So far each theme is name `something-theme`. If we follow this pattern, it describes itself

Reply:
👍 I'll update my terra-core pr updating button and card to the same pattern.